### DISCUSSION TITLE
Dead Player Counter

### DISCUSSION THREAD

**Discussion by Dexteriority** [winstxnhdw/lc-hax#305] [Ideas]
Its like a feature to show on your screen whose name is dead right now.
But do you think this feature important??
Its just my suggestion though xdxd

**Comment by winstxnhdw** [comment]
Oh right, I forgot about this. I was supposed to add this a long time ago.

**Reply by Dexteriority** [reply]
Looking forward for it XD

**Reply by Dexteriority** [reply]
Also! how do i update the lchax i have if you changed/added something to it?? 
I suppose i just download the new updated file from this github?

**Reply by winstxnhdw** [reply]
It auto-updates. Look at the launch.bat file.

**Reply by Dexteriority** [reply]
So the clone taking up original file from server(you) ?
Great!

**Reply by winstxnhdw** [reply]
I am not the server. GitHub is.

**Reply by Dexteriority** [reply]
Oh really? I see!

**Reply by Dexteriority** [reply]
By the way, I don't think my files are updating correctly.
I got this in my launch.bat since you updated some files 2 hours ago:

_error: Pulling is not possible because you have unmerged files.
hint: Fix them up in the work tree, and then use 'git add/rm <file>'
hint: as appropriate to mark resolution and make a commit.
fatal: Exiting because of an unresolved conflict._

I also downloaded newest file manually to use them :((

**Reply by winstxnhdw** [reply]
Yeap, that will happen if you change the code on your end, and we updated ours. That's called a merge conflict. We won't try to reset your changes and leave it up to you to choose whether to keep them or not.

**Reply by Dexteriority** [reply]
Oooooh!!! So i need to keep the file as it is to get the update? 
Do you have suggestion of how to keep my files but the update will keep going? or mabye to make the .bat will tell something like "failed to merge ZZZ.cs because of merge conflict. New Update has been added to ZZZ.cs in github" ?

This is a lot to ask but please feel free to say if its not possible :(

**Reply by winstxnhdw** [reply]
It's not possible. You have to handle the merge conflict yourself with an editor.

**Reply by Dexteriority** [reply]
THANKS!! btw as i said i downloaded the update manually thanks to the merge conflict notification.
But with /sell command, the weight bug still occurs, my character somehow became so heavy that it only moves 1 direction permanently after the typing the command. 
Is it because i forgot to downlaod something, or is it the code missing a line?

**Reply by winstxnhdw** [reply]
You can try these commands to preserve your changes. Not sure how well they will work.

```bash
git pull --rebase --autostash
git checkout --theirs .
git add .
```

> But with /sell command, the weight bug still occurs, my character somehow became so heavy that it only moves 1 direction permanently after the typing the command.

What's your commit hash right now? Do `git rev-parse HEAD^` in the repository.

**Reply by winstxnhdw** [reply]
Nevermind. The logic for the `sell` command was just incorrect. I'll fix it in latest.

**Reply by Dexteriority** [reply]
Okay! glad i could send feedbacks

**Comment by Dexteriority** [comment]
> Nevermind. The logic for the `sell` command was just incorrect. I'll fix it in latest.

hey I'm wondering in which file is the dead player counter located in?

**Reply by winstxnhdw** [reply]
wdym? I haven’t added it yet.

**Reply by Dexteriority** [reply]
oh i thought it was there since there is a big update yesterday right XD

**Reply by winstxnhdw** [reply]
I'll ping this thread when it is added.

**Comment by winstxnhdw** [comment]
In latest, we now notify the player on teammate's death.

**Reply by Dexteriority** [reply]
THANKS!
